src/environmentd/src/http/webhook.rs

```diff
@@ -258,6 +291,11 @@ impl IntoResponse for WebhookError {
             e @ WebhookError::Unavailable => {
                 (StatusCode::SERVICE_UNAVAILABLE, e.to_string()).into_response()
             }
+            e @ WebhookError::TemporarilyUnavailable => {
+                let mut headers = HeaderMap::new();
+                headers.insert(RETRY_AFTER, RETRY_AFTER_VALUE);
+                (StatusCode::SERVICE_UNAVAILABLE, headers, e.to_string()).into_response()
+            }
```

Comment:
Uncovered

Reply:
This error is returned when we hit the race described above, so it's very unlikely to get exercised by a test